libgalois/src/Statistics.cpp

```diff
@@ -250,14 +252,12 @@ katana::StatManager::Print() {
   if (impl_->outfile_.empty()) {
     return PrintStats(std::cout);
   }
+  std::ostringstream out;
+  PrintStats(out);
```

Comment:
A concern here is that the string stream might run out of memory (with some flags like per host and more detailed stats, the stirngs can become quite large). For now, this is fine but please add a NB or TODO

Reply:
Will do.